src/storage-controller/src/instance.rs

```diff
@@ -389,6 +389,7 @@ where
                 command = self.command_rx.recv() => {
                     let Some(mut command) = command else {
                         // Controller is no longer interested in this replica. Shut down.
```

Comment:
Nit: I think the comment is now redundant.

Reply:
clearly! 😅